cms/urls.py

```diff
@@ -89,8 +89,6 @@
     url(r'^request_course_creator$', contentstore.views.request_course_creator, name='request_course_creator'),
     url(r'^course_team/{}(?:/(?P<email>.+))?$'.format(COURSELIKE_KEY_PATTERN),
         contentstore.views.course_team_handler, name='course_team_handler'),
-    url(r'^course_info/{}$'.format(settings.COURSE_KEY_PATTERN), contentstore.views.course_info_handler,
```

Comment:
To be clear, I think I see Production traffic to this URL with 200 responses. Since we already have the flag set to True, maybe this is not something that can go away yet? More investigation required.

Reply:
@robrap Hmm, that makes me think I could cause course_info to redirect to course, but I am unsure of how to do that, or if that is the correct destination of the redirection. @Agrendalath If you have anything to add, please do.

Reply:
Hopefully we don't need a permanent redirect, and we could simply move the calling code to the right location?  That could be a separate minor PR before this lands, and then this could be removed when it isn't in use.

Reply:
@taniwha, should we remove this whole page from Studio?
It allows us to manage updates and course handouts, which are being displayed on the sidebar of the `{course_id}/course/` page in LMS. Without this we won't be able to configure them anymore.

Reply:
@Agrendalath Actually, I've been wondering this myself: the change does seem overly aggressive but I went with "well, this seems to be what was requested".

Reply:
@taniwha, I don't think we should delete this Studio page, as it's still being actively used. That would explain the production traffic for this view.
cc: @marcotuts

Reply:
@Agrendalath I am inclined to agree. Should the decision go that way, I will revert those changes.

Reply:
@taniwha, if we revert this Studio page in one clear commit, then we can remove it again without a problem later. I would suggest to proceed with this approach, as this is still being used.